docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+
+services:
+  database:
+    image: postgres:14.2
+    environment:
+      POSTGRES_USER: root
+      POSTGRES_PASSWORD: 1234
+      POSTGRES_DB: postgres
+    ports:
+      - 5440:5432
```

Comment:
why did you add docker configuration? And how should app be launched now?

Reply:
This is just the configuration for the database. If you don't want to use it, launching the application will be the same.

I added it because I don't want to touch the local database, so I created a docker file for this project. This is useful if you don't have postgres on your computer or don't want to touch it, it will be an isolated, clean database every time you run the container(but when app is launched seeding with liquibase still works for docker DB).